src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
+    const progressNum = + ( ( mainStat / goal ) * 100 ).toFixed(2)
+    return ( progressNum ).toString().concat('%')
+  }
+
+  render() {
+
+    const { title, x, y, mainStat, goal } = this.props
+    const positionStyle = { top: x, left: y }
+    const progressString = this.calculateProgress( mainStat, goal )
+    const isComplete = mainStat / goal >= 1
+
+    return (
+      <article>
+        <div className="widget widget-width-1 widget-height-1" style={ positionStyle }>
+          <Header title={ title }/>
```

Comment:
I missed this in previous submissions - we should leverage `children` in order to make this a little more readable:
```jsx
<Header>{title}</Header>
```
This would mean that the `Header` component should be updated to, in render, use `this.props.children` instead of a `title` prop.